cirq/ops/named_qubit.py

```diff
@@ -23,13 +26,10 @@ class NamedQubit(raw_types.Qid):
     containing `cirq.NamedQubit('qubit22')` and `cirq.NamedQubit('qubit3')`, the
     wire for 'qubit3' will correctly come before 'qubit22'.
     """
-
-    def __init__(self, name: str) -> None:
-        self._name = name
-        self._comp_key = _pad_digits(name)
+    _name: str
 
     def _comparison_key(self):
-        return self._comp_key
+        return _pad_digits(self._name)
```

Comment:
caching this value was likely done for performance reasons. Do you have a sense for the slowdown one would expect from this change?

Reply:
Yes, this caching should not be removed.